api/resources/reminders.py

```diff
@@ -143,32 +118,45 @@ def patch(self, *args, **kwargs):
                 'errors': errors
             }, 400
 
-        if title and len(title.strip()) > 0:
+        if title:
             reminder.title = title
-        reminder.update()
+        if supplies:
+            reminder.supplies = supplies
+        if show_supplies:
+            reminder.show_supplies = show_supplies
+        db.session.commit()
 
-        reminder_payload = _reminder_payload(reminder)
-        reminder_payload['success'] = True
-        return reminder_payload, 200
+        result = reminder_schema.dump(reminder)
+        return result, 200
 
     def delete(self, *args, **kwargs):
-        reminder_id = kwargs['reminder_id']
+        json_data = request.get_json(force=True)
         reminder = None
         try:
-            reminder = db.session.query(Reminder).filter_by(id=reminder_id).one()
+            reminder = Reminder.query.filter_by(id=json_data['id']).first()
         except NoResultFound:
             return abort(404)
 
         reminder.delete()
-        return {}, 204
+        return {'message': 'Reminder successfully deleted'}, 200
+
+class ReminderResource(Resource):
+    def get(self, *args, **kwargs):
```

Comment:
Adding the *args and *kwargs looks like it lets the FE send whatever they want with less restrictions